### ISSUE TITLE
License

### ISSUE THREAD

**Issue by vhscom** [rgossiaux/svelte-headlessui#38]
As far as I'm aware you _cannot_ change ANY existing copyright line in the MIT license. And when you add to an existing license you should stack on top of the existing copyright line (and not below it). As such, nothing in this world is ever set in stone. I'm simply relaying what I've read and researched on opensource.stackexchange.com and on the web. Lastly you may want to update your modifications copyright line to include something like _and contributors_ if you plan to accept pull requests in this repo.

Thanks for picking an MIT license. Even if you did go with Apache 2.0 (which you possibly still could do if you wanted, though that's just a hip shot statement) both the MIT and Apache 2.0 are GPL-compatible licenses.

**Comment by rgossiaux** [comment]
@vhscom Just curious, why'd you open this issue & then close it?

**Comment by vhscom** [comment]
In hopes of adding a little value without causing a distraction.

**Comment by vhscom** [comment]
In hopes of adding a little value without causing a distraction. I believe Discussions will be of great value in that regard. Thanks again for this creative work.

**Comment by vhscom** [comment]
Current license is not recognized by GitHub which means other parsers may have problems scanning it. It's best if the license can be seen in the repo homepage to save the user time and ensure the license text is fully intact.

**Comment by vhscom** [comment]
![Screenshot from 2022-01-29 01-37-41](https://user-images.githubusercontent.com/97140109/151595082-d8555e62-8f39-4946-a32b-282f42f21ecc.png)

Current license is not recognized by GitHub which means other parsers may have problems scanning it. It's best if the license can be seen in the repo homepage to save the user time and ensure the license text is fully intact. It should say MIT License in this image.

**Comment by vhscom** [comment]
![Screenshot from 2022-01-29 01-38-48](https://user-images.githubusercontent.com/97140109/151595293-30252079-86f6-4711-aa4a-64d2c5ebfcc7.png)

Current license is not recognized by GitHub which means other parsers may have problems scanning it, not to mention us humans. When formatted properly the license will appear as MIT License in the area pictured above.

**Comment by rgossiaux** [comment]
Ok, I finally fixed this. Thanks for the nudge. Originally I had tried using multiple copyright lines, but the GH license detection didn't work. Turns out every copyright line needs to begin with `Copyright` for GH to parse it correctly.

I added the "and contributors" line per your suggestion, though I'm not too concerned about the order of the copyright lines--there's nothing in the license that says they have to be in a particular order, just that the original copyright line is preserved.
